probe/host/system_linux.go

```diff
@@ -31,6 +33,67 @@ var GetKernelReleaseAndVersion = func() (string, string, error) {
 	return string(release), string(version), nil
 }
 
+// GetPrettyName extracts the PRETTY_NAME from host's /etc/os-release
+// SPEC: https://www.freedesktop.org/software/systemd/man/os-release.html
+var GetPrettyName = func () (string, error) {
+
+	var buffer bytes.Buffer
+
+	buf, err := ioutil.ReadFile("/var/run/scope/host-os-release")
+	if err != nil {
+		return runtime.GOOS, nil
+	}
+
+	// If can't find host-os-release, get container's os-release
```

Comment:
Nope. We really don't want to surface the scope container base OS as the node OS :)